.github/workflows/main-build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         with:
           path: |
             ~/.m2/repository/io/opentelemetry/
-          key: ${{ runner.os }}-maven-local-${{ hashFiles('.github/patches/opentelemetry-java-*.patch') }}
+          key: ${{ runner.os }}-maven-local-${{ hashFiles('.github/patches/opentelemetry-java*.patch') }}
```

Comment:
Did you do a code search and verify that you changed all instances of this cache key?

Reply:
yes I changed in both pr and main build. I don't see this hash key anywhere else in the code - did the search.